roles/base/tasks/redhat_tasks.yml

```diff
@@ -10,6 +10,7 @@
     update_cache: true
     state: latest
   with_items:
+    - gcc
```

Comment:
What do you guys want me to do? gcc is required to build both netplugin 
and volplugin, yet the dev role will absolutely not work with the centos 
vm.

-Erik